### PR TITLE
New test branch

### PR DESCRIPTION
**Pull request by HaroldBernard**

### CODE REVIEW

File: shoreline.go [hunk 1]
```diff
@@ -142,32 +117,39 @@ func main() {
...
+		log.Fatalln(err)
+	}
+
+	notifier, err := user.NewUserEventsNotifier(kafkaConfig)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	handler, err := user.NewUserEventsHandler(clientStore)
+	if err != nil {
+		log.Fatalln()
```
Comment (derrickburns) [inline shoreline.go:144]: Add error messsage.

File: user/api.go [hunk 2]
```diff
@@ -376,18 +367,12 @@ func (a *Api) UpdateUser(res http.ResponseWriter, req *http.Request, vars map[st
...
-					} else {
-						a.marketoManager.UpdateListMembershipForUser(originalUser, updatedUser)
-					}
-				} else {
-					failedMarketoUploadCount.Inc()
-				}
-			}
 			a.logMetricForUser(updatedUser.Id, "userupdated", sessionToken, map[string]string{"server": strconv.FormatBool(tokenData.IsServer)})
+			if err := a.userEventsNotifier.NotifyUserUpdated(req.Context(), *originalUser, *updatedUser); err != nil {
+				a.logger.Println(http.StatusInternalServerError, err.Error())
```
Comment (derrickburns) [inline user/api.go:372]: Can we add a prometheus counter for this error?

**Review by derrickburns — CHANGES_REQUESTED** [review]
Just a few small issues.

File: shoreline.go [hunk 3]
```diff
@@ -142,32 +117,39 @@ func main() {
...
+	}
+	consumer, err := events.NewSaramaCloudEventsConsumer(kafkaConfig)
+	if err != nil {
+		log.Fatalln()
```
[the remark this answers is not in the document]
Comment (HaroldBernard) [reply shoreline.go]: done

File: shoreline.go (hunk 1 above)
Comment (HaroldBernard) [reply shoreline.go]: done

File: user/api.go (hunk 2 above)
Comment (HaroldBernard) [reply user/api.go:376]: done

**Review by derrickburns — APPROVED** [review]
Lgtm

**Comment by HaroldBernard** [comment]
retested with master merge and is good to go

**Review by derrickburns — APPROVED** [review]
LGTM

**Review by toddkazakov — APPROVED** [review]
LGTM

**Pull request merged**